scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
@mgax I don't think the gmtime conversion is bad (except for maybe craziness with leap seconds?). Will this work?
```python
logging.Formatter.converter = lambda: datetime.utcnow().timetuple()
```
And if needed we can customize localsettings to use localtime and make `LOG_DATEFORMAT` use `%Z` instead on dev. The alternative for doing it properly is writing a custom converter that does really proper timezone conversion based on the app's timezone setting, but... we already spent too much effort on this (sorry btw, I started it). :)

Anyway, everything should output and store timezone-aware times so we preserve our sanity.